pkg/plugins/utilities.go

```diff
@@ -135,16 +135,68 @@ func RefreshPluginManifest(ctx context.Context, config config.IConfig, fs afero.
 	return nil
 }
 
-func validatePluginManifest(body []byte) error {
+func fetchAndMergeManifests(pluginData requests.PluginData) (*PluginList, error) {
+	pluginList, err := fetchPluginList(pluginData.PluginBaseURL, "plugins.toml")
+	if err != nil {
+		return nil, err
+	}
+
+	additionalPluginLists := []*PluginList{}
+	for _, filename := range pluginData.AdditionalManifests {
+		additionalPluginList, err := fetchPluginList(pluginData.PluginBaseURL, filename)
+		if err != nil {
+			return nil, err
+		}
+		additionalPluginLists = append(additionalPluginLists, additionalPluginList)
+	}
+
+	mergePluginLists(pluginList, additionalPluginLists)
+
+	return pluginList, nil
+}
+
+func fetchPluginList(baseURL, manifestFilename string) (*PluginList, error) {
+	pluginManifestURL := fmt.Sprintf("%s/%s", baseURL, manifestFilename)
+	body, err := FetchRemoteResource(pluginManifestURL)
+	if err != nil {
+		return nil, err
+	}
+	return validatePluginManifest(body)
+}
+
+// Merge additional plugin lists into the main plugin list, in place
+func mergePluginLists(pluginList *PluginList, additionalPluginLists []*PluginList) {
+	for _, list := range additionalPluginLists {
+		for _, pl := range list.Plugins {
+			idx := findPluginIndex(pluginList, pl)
+			if idx == -1 {
+				pluginList.Plugins = append(pluginList.Plugins, pl)
+			} else {
+				pluginList.Plugins[idx].Releases = append(pl.Releases, pluginList.Plugins[idx].Releases...)
```

Comment:
Just realized this line violates this assumption: https://github.com/stripe/stripe-cli/blob/fdf82762ad5089ebc4ce0839b9401093a1c58495/pkg/plugins/plugin.go#L148

I will update this to sort the releases after merging.